bin/configs/typescript-node-default.yaml

```diff
@@ -1,4 +1,4 @@
 generatorName: typescript-node
 outputDir: samples/client/petstore/typescript-node/default
-inputSpec: modules/openapi-generator/src/test/resources/2_0/petstore.yaml
```

Comment:
Three ideas:

1. use a new config for this, so the old fixture stays intact (it even has a different spec version, so there might be other things that are tested by using it)
2. add a deprecated tag to the spec (if 2.0 supports it)
3. add a deprecated tag to a different 3.0 spec

2) and 3) also have the advantage that there is no new spec added to the repo.

Reply:
Pushed a change with #1 to generate samples with a 3.0 file